src/test/java/io/stargate/sgv2/jsonapi/api/v1/DeleteManyIntegrationTest.java

```diff
@@ -253,17 +257,69 @@ public void noFilter() {
           .body("errors", is(nullValue()));
     }
 
+    /**
```

Comment:
Reword: can basically say that this tests same behavior as `noFilter()` but with "empty" filter.